### PR TITLE
Expose an environmental value to virtio win image

### PR DESCRIPTION
**Pull request by tiraboschi**
Expose an environmental value to consume at runtime
a virtio win image that should be propagated to
a config map consumed by the UI component
when the user tries to boot/import a Windows VM.

In the past the UI was using an hardcoded value
but due to that it wasn't properly working on
disconnected environments.

Fixes: https://bugzilla.redhat.com/1942839

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:
```release-note
Expose an environmental value to virtio win image
```

### CODE REVIEW

File: deploy/images.csv [hunk 1]
```diff
@@ -1,9 +1,11 @@
 image_var,name,tag,digest
+<<<<<<< HEAD
```
Comment (yaacov) [inline deploy/images.csv:2]: typo ?

Comment (tiraboschi) [reply deploy/images.csv:2]: yes, thanks!

**Pull request merged**